enterprise/frontend/src/metabase-enterprise/metabot/components/MetabotChat/utils.ts

```diff
@@ -0,0 +1,5 @@
+// NOTE: JSDOM does not implement the `scrollTo` method on elements
+// so this function is used for mocking purposes in Jest
+export function scrollTo(element: HTMLDivElement, options: ScrollToOptions) {
```

Comment:
Can we put this in `frontend/test/__support__/mocks.js` instead? There we have scrollBy and scrollIntoView as well

Reply:
Ah, I think I can use what already exists in that file. Thanks!

Reply:
Good call! Thanks.